### DISCUSSION TITLE
identify or take over the information from external music source

### DISCUSSION THREAD

**Discussion by MRi-LE** [music-assistant/support#1522] [Q&A]
Hello,

i am new to Music Assistant and love the integration especially that it makes spotcast & scripts obsolete in my config.

Now i want MA to identify or take over the role of external music source.

Eg.

1.) I use mobile and play Spotify ( Spotify Connect )
2.) I use TuneIn via Google Chrome ( cloud )

Whereas the default media player list the content whats being played right now, the MA just shows external soure and no further info.

is there a way to connect it somehow?

The only alternative i can think of is, to create automations in google home to always call MA to trigger the Addon ( Spotify or TuneIn ) but this doesnt help if someone uses Spotify Connect at home

**Comment by OzGav** [comment]
If you are going to use MA then use it solely. The UI is getting better all the time.
